ansible/roles/postgres/vars/main/main.yml

```diff
@@ -163,6 +178,8 @@ postgres_databases:
     owner: metabase
   - name: pixels
     owner: pixels
+  - name: devops
```

Comment:
Why do you need a database for devops ?

Reply:
It's part of the configuration needed for mTLS as far as I am aware, the CN must match the database at least for the initial connection.

However, on top of that I think that given the account is a superuser connecting to a dummy zero-use database is a better idea than defaulting to something with more privilege/importance. I'd rather have people connect and then change database.